### PR TITLE
feature: add ignoreUrlPatterns to TrackingExtensionExisting

### PR DESCRIPTION
**Pull request by 4rthurmonteiro**
ISSUE: https://github.com/DataDog/dd-sdk-flutter/issues/614
### What and why?

To filter http tracking when using Flutter module.

What changes does this pull request introduce and why is it necessary?

Add ignoreUrlPatterns to TrackingExtensionExisting extension

### How?

Adding in enableHttpTracking the attribute ignoreUrlPatterns.

If needed, a description of how this PR accomplishes what it does.

### Review checklist

- [x] This pull request has appropriate unit and / or integration tests 
- [x] This pull request references a Github or JIRA issue

### CODE REVIEW

**Comment by 4rthurmonteiro** [comment]
@fuzzybinary i open this PR based on https://github.com/DataDog/dd-sdk-flutter/pull/613 with the signed commits.

File: packages/datadog_tracking_http_client/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 ## Unreleased
 
+## 2.2.1
```
Comment (fuzzybinary) [inline packages/datadog_tracking_http_client/CHANGELOG.md]: Leave it under the `Unreleased` heading.  We'll add the version when we release it.

Comment (4rthurmonteiro) [reply packages/datadog_tracking_http_client/CHANGELOG.md]: done

**Review by fuzzybinary — APPROVED** [review]

**Comment by 4rthurmonteiro** [comment]
@fuzzybinary the bitrise check failed. What I can do to fix it?

**Pull request merged**